spec/spec_helper.rb

```diff
@@ -11,35 +12,29 @@
 require 'napa'
 require 'napa/rspec_extensions/response_helpers'
 
+module NapaSpecClassHelpers
+  # Apparently Thor's last name is Odinson... who knew?!
+  # http://marvel.com/universe/Thor_(Thor_Odinson)
+  # http://www.infinitelooper.com/?v=4D7cPH7DHgA#/170;230
+  def tell_me_mr_odinson_what_good_is_a_phone_call_if_youre_unable_to_speak?
```

Comment:
what a weird name!

But, this method should probably be called `slience_thor`. Or named whatever the actual function is doing.